scripts/build.sh

```diff
@@ -50,11 +50,13 @@ ${GO_CMD} build \
 
 # Move all the compiled things to the $GOPATH/bin
 OLDIFS=$IFS
-IFS=: MAIN_GOPATH=($GOPATH)
+IFS=: FIRST=($GOPATH) BIN_PATH=${GOBIN:-${FIRST}/bin}
 IFS=$OLDIFS
 
-rm -f ${MAIN_GOPATH}/bin/vault
-cp bin/vault ${MAIN_GOPATH}/bin/
+# Ensure the go bin folder exists
+mkdir -p ${BIN_PATH}
```

Comment:
I think I have been hit by this before in some weird environment too, so thank you for fixing this!

Reply:
Yeah, it's minor issue but a nice fix for users, particularly if they aren't as familiar with Go's paths.